hazelcast/src/test/java/com/hazelcast/config/XMLConfigBuilderTest.java

```diff
@@ -147,79 +124,33 @@ public void testSecurityInterceptorConfig() {
-        System.setProperty("hazelcast.config", "classpath:idontexist");
-        new XmlConfigBuilder();
-    }
-
-    @Test
-    public void loadingThroughSystemProperty_existingClasspathResource() throws IOException {
-        System.setProperty("hazelcast.config", "classpath:test-hazelcast.xml");
-
-        XmlConfigBuilder configBuilder = new XmlConfigBuilder();
-        Config config = configBuilder.build();
-        assertEquals("foobar", config.getGroupConfig().getName());
-    }
-
-    @Test
-    public void testCleanNodeName() {
-        XmlConfigBuilder configBuilder = new XmlConfigBuilder();
-        assertEquals("nocolon", configBuilder.cleanNodeName("noColon"));
-        assertEquals("after", configBuilder.cleanNodeName("Before:After"));
-        assertNull(configBuilder.cleanNodeName((String) null));
-    }
-
-    @Test
     public void readAwsConfig() {
         String xml =
-                "<hazelcast xmlns=\"http://www.hazelcast.com/schema/config\">\n" +
-                        "   <group>\n" +
-                        "        <name>dev</name>\n" +
-                        "        <password>dev-pass</password>\n" +
-                        "    </group>\n" +
-                        "    <network>\n" +
-                        "        <port auto-increment=\"true\">5701</port>\n" +
-                        "        <join>\n" +
-                        "            <multicast enabled=\"false\">\n" +
-                        "                <multicast-group>224.2.2.3</multicast-group>\n" +
-                        "                <multicast-port>54327</multicast-port>\n" +
-                        "            </multicast>\n" +
-                        "            <tcp-ip enabled=\"false\">\n" +
-                        "                <interface>127.0.0.1</interface>\n" +
-                        "            </tcp-ip>\n" +
-                        "            <aws enabled=\"true\" connection-timeout-seconds=\"10\" >\n" +
-                        "                <access-key>access</access-key>\n" +
-                        "                <secret-key>secret</secret-key>\n" +
-                        "            </aws>\n" +
-                        "        </join>\n" +
-                        "        <interfaces enabled=\"false\">\n" +
-                        "            <interface>10.10.1.*</interface>\n" +
-                        "        </interfaces>\n" +
-                        "    </network>\n" +
-                        "</hazelcast>";
+                HAZELCAST_START_TAG +
+                "   <group>\n" +
+                "        <name>dev</name>\n" +
+                "        <password>dev-pass</password>\n" +
+                "    </group>\n" +
+                "    <network>\n" +
+                "        <port auto-increment=\"true\">5701</port>\n" +
+                "        <join>\n" +
+                "            <multicast enabled=\"false\">\n" +
```

Comment:
We should really get this test class split up so that every data-structure like IQueue, IMap gets its own test file.